### PR TITLE
Atomic/containers.py: Add NAME field to container list

### PR DESCRIPTION
**Pull request by baude**
## Description
Issue #1021 requested we add the NAME field to the containers list command,
so that debugging container failures would be easier.

## Related Bugzillas
-
-

## Related Issue Numbers
- #1021 
-

## Pull Request Checklist:

If your Pull request contains new features or functions, tests are required. If the PR is a bug fix and no tests exist, please consider adding some to prevent regressions.
- [ ] Unittests
- [ ] Integration Tests

### CODE REVIEW

**Comment by yuqi-zhang** [comment]
Since everything got truncated a bit more the whole field of: https://github.com/projectatomic/atomic/blob/master/tests/integration/test_system_containers_runtime.sh#L73 is no longer being displayed. Could just truncate that to match "/usr/bin" for now.

**Comment by baude** [comment]
I'm actually going to add --no-trunc to the containers list call ... that fixed it.

**Comment by yuqi-zhang** [comment]
Well the point was to test the flagless call first (just `atomic containers list`), --no-trunc is tested later as a separate case. But I guess its not a big issue. I trust your judgement call.

**Comment by baude** [comment]
ah nice catch, i will fix it!

**Comment by yuqi-zhang** [comment]
The max_command field seems to be mis-populated? take etcd for example:
https://paste.fedoraproject.org/paste/unSmdlxbWzSEfaUxJZXOb15M1UNdIGYhyRLivL9gydE=

File: tests/integration/test_system_containers_runtime.sh [hunk 1]
```diff
@@ -70,7 +69,8 @@ sleep 0.5s
 # 3. Test container list functionality
 systemctl start ${NAME}.service
 ${ATOMIC} containers list > ${WORK_DIR}/ps.out
-assert_matches "/usr/bin/run.sh" ${WORK_DIR}/ps.out
+cat ${WORK_DIR}/ps.out
```
Comment (ashcrow) [inline tests/integration/test_system_containers_runtime.sh]: Leftover from debugging?

**Comment by ashcrow** [comment]
@yuqi-zhang Confirmed. I get the same output with a1dd1bc7b76aa2406c247b215e58f7d3b9e41fc5 applied.

```
# atomic containers list
   CONTAINER ID IMAGE                NAME       COMMAND    CREATED          STATE      BACKEND    RUNTIME   
   test         docker.io/gscrivano/ test       /usr/bin/e 2017-06-06 14:26 running    ostree     runc      
#  atomic containers list --no-trunc
   CONTAINER ID IMAGE                    NAME                               COMMAND CREATED          STATE      BACKEND    RUNTIME   
   test docker.io/gscrivano/etcd test                               /usr/bin/etcd-env.sh /usr/bin/etcd 2017-06-06 14:26 running    ostree     runc      
# atomic containers list -a --no-trunc
   CONTAINER ID IMAGE                    NAME                               COMMAND CREATED          STATE      BACKEND    RUNTIME   
   test docker.io/gscrivano/etcd test                               /usr/bin/etcd-env.sh /usr/bin/etcd 2017-06-06 14:26 running    ostree     runc 
```

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by baude** [comment]
@ashcrow @yuqi-zhang mispopulated == columns not lining up ?

**Comment by ashcrow** [comment]
@baude yes. With ``--no-trunc`` the resulting data did not line up. @yuqi-zhang's paste timed out but [this](https://github.com/projectatomic/atomic/pull/1024#issuecomment-306576714) shows the same basic result.

**Comment by ashcrow** [comment]
5b12525 produces the expected result.

**Comment by baude** [comment]
@rh-atomic-bot r+  5b12525

**Pull request closed without merging**